cluster/gce/util.sh

```diff
@@ -878,7 +878,7 @@ function construct-windows-kubelet-flags {
   flags+=" --logtostderr=false"
 
   # Configure the file path for host dns configuration
-  flags+=" --resolv-conf=${WINDOWS_CNI_CONFIG_DIR}\hostdns.conf"
+  flags+=" --resolv-conf=${WINDOWS_CNI_DIR}\hostdns.conf"
```

Comment:
this commit is still here, could you move it to a separate pr?

Reply:
See the comment https://github.com/kubernetes/kubernetes/pull/85396#issuecomment-559217726